src/store/selectors.ts

```diff
@@ -0,0 +1,15 @@
+import { createSelector } from 'reselect';
+
+import { getFarmerDairyBalanceNoFormat as getFarmerDairyBalance } from './modules/milk/selectors';
+import { getFarmerLoanBalance } from './modules/loan/selectors';
+import { getFarmerPaymentBalance } from './modules/payment/selectors';
+
+/**
+ * Selector for the farmer total balance for the week
+ * Includes payments, loans, and dairy transactions
+ */
+export const getFarmerTotalBalance = createSelector(
+  [getFarmerPaymentBalance, getFarmerLoanBalance, getFarmerDairyBalance],
+  (paymentBalance: string, loanBalance: string, dairyBalance: string) => 
+  ((Number(dairyBalance) - Number(paymentBalance) - Number(loanBalance)).toString()),
```

Comment:
Looks good only problem I can see is that the decimal on the balance throws the page formatting off. could you implement that formatter selector function to truncate the decimals and add the K suffix to large numbers.

Reply:
Absolutely

Reply:
Implemented. I also tweaked the formatter to allow for large negative numbers as well.